rdt-open-reader/src/covidIndonesia/java/io/ona/rdt/activity/CovidPatientHistoryActivity.java

```diff
@@ -43,6 +43,10 @@ protected void onCreate(Bundle savedInstanceState) {
         currPatient = getIntent().getParcelableExtra(Constants.FormFields.PATIENT);
         presenter = new CovidPatientHistoryActivityPresenter(this);
 
+        TextView tv = findViewById(R.id.visit_date_label);
+        String visitDateLabel = getString(R.string.visit_date_label);
+        tv.setText(visitDateLabel + ": " + getIntent().getStringExtra(Constants.FormFields.PATIENT_VISIT_DATE));
```

Comment:
We can move `getIntent().getStringExtra(Constants.FormFields.PATIENT_VISIT_DATE))` to a global variable, it's used twice in the file.

Reply:
You mean instance variable? Done.